.github/workflows/codeql-analysis.yml

```diff
@@ -48,6 +48,11 @@ jobs:
           # Defining a fixed CodeQL bundle version
           tools: https://github.com/github/codeql-action/releases/download/codeql-bundle-20230207/codeql-bundle-linux64.tar.gz
 
+      - name: Set Swap Space
+        uses: pierotofy/set-swap-space@master
```

Comment:
Are we fine with using an unpinned version of a 3rd-party GitHub Action? This could break without warning, so I'd at least pin it to a version / commit we manually vetted.

Reply:
Thanks for the suggestion @KSerrania. I updated by pinning to the latest commit on its master branch.